test/e_chronicler/models/journal_entry_test.exs

```diff
@@ -83,4 +83,11 @@ defmodule EChronicler.Models.JournalEntryTest  do
     assert {:error, %Ecto.Changeset{}} = JournalEntry.update_journal_entry(journal_entry, @invalid_journal_attrs)
   end
 
+  test "delete_journal_entry/1 deletes the journal_entry" do
+    assert {:ok, %JournalEntry{} = journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(@valid_journal_attrs)
+    assert {:ok, %JournalEntry{}} = JournalEntry.delete_journal_entry(journal_entry)
+    assert JournalEntry.get_journal_entry(journal_entry.id) == nil
+  end
```

Comment:
If you amend `delete_journal_entry` as I recommended, you'll need a test for the error case of the entry ID not existing.

Reply:
I'm having trouble capturing the error case in a test, I'm getting this when I put in a bad ID in the test:
`   ** (FunctionClauseError) no function clause matching in Ecto.Repo.Schema.delete/4`

But I'm unable to capture that in my current test, which is this: 
```
  test "delete_journal_entry/1 with invalid id returns error" do
    assert {:error, FunctionClauseError} = JournalEntry.delete_journal_entry(1000)
  end
```

I'm sure part of that is because the error is thrown by Ecto, not by the `delete_journal_entry` function itself - so is the move to add cases for the interior `Repo.get(JournalEntry, id)` call?